_integration-schemas/selligent/transactional_mailing.md

```diff
@@ -0,0 +1,75 @@
+---
+tap: "selligent"
+version: 1.0
+key: ""
+
+name: "transactional_mailing"
+doc-link: 
+singer-schema: https://github.com/singer-io/tap-selligent/blob/master/tap_selligent/schemas.py
+description: "This table contains transactional mailing data."
+
+
+replication-method: "Full Table"
+
+replication-key:
+  name: ""
+
+api-method:
+  name: ""
+  doc-link: ""
+
+attributes:
+  - name: "id"
+    type: "integer"
+    primary-key: true 
+    description: "The asset_id of this mailing."
+    foreign-key-id: "transactional-id"
```

Comment:
Just like with Klaviyo, let's comment this out for now since it doesn't point to anything.